docs/content/en/docs/reference/changelog.md

```diff
@@ -6,7 +6,18 @@ weight: 35
 
 ## Unreleased
 
+### Added
```

Comment:
we also added configurability for MHC timeouts with an env var input

Reply:
Do you have an issue link or PR?

Reply:
https://github.com/aws/eks-anywhere/pull/4123/files

Reply:
Done, let me know